features/step_definitions/basic_steps.rb

```diff
@@ -8,6 +8,11 @@
     end
 end
 
+Given("I am logged in as {string}") do |name|
+  user = User.find_by(name: name)
+  login_as(user, scope: :user)
```

Comment:
can u try writing this is in poetry mode

Reply:
done!